src/ontology/mondo-edit.obo

```diff
@@ -250216,7 +250216,7 @@ is_a: MONDO:0018054 {source="DC-OMIM:611494", source="MONDOLEX:0012678", source=
 [Term]
 id: MONDO:0012679
 name: autosomal recessive osteopetrosis 6
-def: "Any autosomal recessive malignant osteopetrosis in which the cause of the disease is a mutation in the CLCN7 gene." [MONDO:patterns/disease_series_by_gene]
+def: "Any autosomal recessive malignant osteopetrosis in which the cause of the disease is a mutation in the PLEKHM1 gene." [MONDO:patterns/disease_series_by_gene]
```

Comment:
good catch. I don't know how this happened, but we need to have automated checks to prevent this, @matentzn